draft-ietf-scitt-architecture.md

```diff
@@ -297,7 +297,8 @@ Transparency is implemented by providing a consistent, append-only, cryptographi
 A SCITT instance is referred to as a Transparency Service.
 Implementations of Transparency Services may protect their Append-only Log using a combination of trusted hardware, replication and consensus protocols, and cryptographic evidence.
 A Receipt is an offline, universally-verifiable proof that an entry is recorded in the Append-only Log.
-Receipts do not expire, but it is possible to append new entries (more recent Signed Statements) that subsume older entries (less recent Signed Statements).
+Receipts do not have an expiration time, but the corresponding key material could.
+Requesting a refreshed receipt can result in the same receipt with a different signature or signature with a different signing algorithm.
```

Comment:
Since the above  (line 300) refers to expired key material, should this line reflect an updated key material expiration?

```suggestion
Receipts do not have an expiration time, but the corresponding key material could.
Requesting a refreshed receipt can result in a new receipt over the same signed statement, with an updated signature, signing algorithm and new key material expiration.
```

Reply:
How does a Client request a refreshed Receipt? Can it perform this operation without re-registering the same Signed Statement as before?

Reply:
The details of how to request a receipt is and should be covered in SCRAPI.

To Steve's comment, I don't think this is what we want to say.  The content we are re-issuing is based on the entries in the Ledger and using the receipt to find that.  There is no presentation or look up of signed statement so this leads people to a wrong point of view.  Yes the signed statement is then indirectly covered by the re-issuance.  The important point that we lose with this proposal is the epoch we face with PQC.  We can generate new receipts with PQC algorithms without having to re-sign the original document.  Which goes to the signature is preserved and valid at the time we generated a receipt, the fact that there are stronger signing algorithms does not mean that old signatures are suspect.

Reply:
> The details of how to request a receipt is and should be covered in SCRAPI.

I can agree to that, but the mechanics of this are part of the problem. I would support rewording this sentence if it said specifically what a client must do to refresh a receipt. Otherwise, I would think it is best to leave this out. Is the ability to refresh a receipt a mandatory, essential feature of every TS that must conform to this architecture? It is hard for me to know, because what must be done.

> The important point that we lose with this proposal is the epoch we face with PQC. We can generate new receipts with PQC algorithms without having to re-sign the original document. 

Re my previous point, that is why it is hard for me to understand the PQC implications and why it is essential all spec-conformant TS implementations need the ability to reissue receipts without the original document. This approach seems interesting but goes beyond the minimally viable architecture requirement _for everyone_, no?

Reply:
> The important point that we lose with this proposal is the epoch we face with PQC. We can generate new receipts with PQC algorithms without having to re-sign the original document. Which goes to the signature is preserved and valid at the time we generated a receipt, the fact that there are stronger signing algorithms does not mean that old signatures are suspect.

@roywill, when you say "re-issue receipts", without re-signing the original document, I'm assuming this is a new receipt for the same registration of the signed statement (original document). Wouldn't that be a new notary signature from the TS, using new algorithms and a new expiration date on the notary signature? It would evaluate the signed statement against the same registration policy, not a new one. (I'm avoiding the counter-signature reference). 

 ```suggestion
Receipts do not have an expiration time, but the corresponding key material could.
Requesting a refreshed receipt can result in a new receipt over the previously signed statement, with an updated signature, signing algorithm, and new key material expiration.
```